elements/pl_file_editor/pl_file_editor.mustache

```diff
@@ -0,0 +1,25 @@
+{{#question}}
+<script>
+$(function() {
+    var UUID = "{{uuid}}";
```

Comment:
I like this UUID system. It's kind of hacky, but in a really straightforward, explicit, and robust way.